<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs5</class>
 <widget class="QWidget" name="IDs5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 5 TID/SID</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileMACAddress">
        <property name="text">
         <string>MAC Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileTimer0Value">
        <property name="text">
         <string notr="true">621-621</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileMACAddressValue">
        <property name="text">
         <string notr="true">9BF123456</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileDSTypeValue">
        <property name="text">
         <string>DS Original/Lite</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="labelProfileKeypressesValue">
        <property name="text">
         <string>None, 1, 2, 3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileDSType">
        <property name="text">
         <string>DS Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileVCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="10" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileTimer0">
        <property name="text">
         <string>Timer0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileGxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileGxStatValue">
        <property name="text">
         <string notr="true">6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileVCountValue">
        <property name="text">
         <string notr="true">2F</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileVFrame">
        <property name="text">
         <string>VFrame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>White</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileVFrameValue">
        <property name="text">
         <string notr="true">5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelProfileKeypresses">
        <property name="text">
         <string>Keypresses</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxSeedFinder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Seed Finder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelSecondRange">
        <property name="text">
         <string>Second Range</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="TextBox" name="textBoxSeedFinderMaxAdvances">
        <property name="text">
         <string notr="true">100</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelAdvanceRange">
        <property name="text">
         <string>Max Advances</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxMinute">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBoxMaxSecond">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxHour">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelHour">
        <property name="text">
         <string>Hour</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxMinSecond">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="TextBox" name="textBoxSeedFinderTID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDate">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelMinute">
        <property name="text">
         <string>Minute</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPushButton" name="pushButtonFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkBoxSeedFinderExistingSave">
        <property name="text">
         <string>Existing save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="TableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxSearchBy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Search By</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxPID">
        <property name="text">
         <string>PID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="TextBox" name="textBoxPID"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="TextBox" name="textBoxTID"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="TextBox" name="textBoxSID"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDateTime">
        <property name="text">
         <string>Start/End Date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEditStart">
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDateEdit" name="dateEditEnd">
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxExistingSave">
        <property name="text">
         <string>Existing save</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelMaxFrame">
        <property name="text">
         <string>Max Advances</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="TextBox" name="textBoxMaxAdvances">
        <property name="text">
         <string notr="true">100</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>checkBoxPID</tabstop>
  <tabstop>textBoxPID</tabstop>
  <tabstop>checkBoxTID</tabstop>
  <tabstop>textBoxTID</tabstop>
  <tabstop>checkBoxSID</tabstop>
  <tabstop>textBoxSID</tabstop>
  <tabstop>checkBoxExistingSave</tabstop>
  <tabstop>textBoxMaxAdvances</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>textBoxSeedFinderTID</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>spinBoxHour</tabstop>
  <tabstop>spinBoxMinute</tabstop>
  <tabstop>spinBoxMinSecond</tabstop>
  <tabstop>spinBoxMaxSecond</tabstop>
  <tabstop>textBoxSeedFinderMaxAdvances</tabstop>
  <tabstop>checkBoxSeedFinderExistingSave</tabstop>
  <tabstop>pushButtonFind</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
